src/app/components/applications/SSV/RegisterValidatorHome/components/OfflineKeyShareGeneration/OfflineKeyShareGeneration.tsx

```diff
@@ -31,10 +32,8 @@ const OfflineKeyShareGeneration = () => {
     const {
         operatorsIds,
         operatorsKeys,
-    } = Object.values(operatorStore.selectedOperators).sort((a: any, b: any) => a.id - b.id).reduce((aggr, operator: IOperator) => {
-        // @ts-ignore
+    } = Object.values(operatorStore.selectedOperators).sort((a: any, b: any) => a.id - b.id).reduce((aggr: any, operator: IOperator) => {
```

Comment:
This is weird part of the code, let's simplify it. First sort it to operators array constant. Then extract ids and keys in next code block

Reply:
done